### PR TITLE
fix: throw err when supplied config is absent

### LINKED ISSUE #1759: [Bug]: Throw error when config file is absent
**Describe the bug**

When user states which config to use and we don't find it we don't throw err and rather fallback to default config/

**What is the current behavior?**

It should throw error when config is absent, it can use default config when config flag is absent

**To Reproduce**
webpack -c webpack.absent-config.js
Bundle succeeds

**Expected behavior**
It should throw err 

**Screenshots**
NA

**Please paste the results of `webpack-cli info` here, and mention other relevant information**
NA

**Additional context**

/cc @evilebottnawi

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

fix

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Should we state this somewhere?

**Summary**

- throw err when requested config file is absent

**Does this PR introduce a breaking change?**
No

**Other information**
Fixes #1759

### CODE REVIEW

**Review by evenstensberg — COMMENTED** [review]
Remove the processingMessageBuffer when you're at it

**Comment by anshumanv** [comment]
Yeah we should get rid of all instances, unused

File: test/config/absent/config-absent.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const { run } = require('../../utils/test-utils');
+
+describe('Config:', () => {
+    it('supplied config file is absent', () => {
+        const { stdout, stderr, exitCode } = run(__dirname, ['-c', resolve(__dirname, 'webpack.config.js')], false);
+        // should throw with correct exit code
+        expect(exitCode).toBe(2);
+        expect(stdout).toBeFalsy();
+        // Should contain the correct error message
+        expect(stderr).toContain("ConfigError: The specified config file doesn't exist in");
```
Comment (evilebottnawi) [inline test/config/absent/config-absent.test.js:13]: Let's add config filename to error

Comment (anshumanv) [reply test/config/absent/config-absent.test.js]: Done 💯

**Comment by anshumanv** [comment]
Done @evenstensberg 💯

**Review by evenstensberg — APPROVED** [review]

**Review by evilebottnawi — APPROVED** [review]

**Comment by evilebottnawi** [comment]
Good job!

**Review by rishabh3112 — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Pull request merged**